client-src/elements/chromedash-enterprise-release-notes-page.ts

```diff
@@ -13,28 +13,50 @@ import {
   parseRawQuery,
   renderHTMLIf,
 } from './utils.js';
+import {customElement, state} from 'lit/decorators.js';
 
 const milestoneQueryParamKey = 'milestone';
 
-export class ChromedashEnterpriseReleaseNotesPage extends LitElement {
-  static get properties() {
-    return {
-      currentFeatures: {type: Array},
-      upcomingFeatures: {type: Array},
-      features: {type: Array},
-      channels: {type: Object},
-      selectedMilestone: {type: Number},
-    };
-  }
+export interface Feature {
```

Comment:
Please don't add multiple Feature interfaces. Instead, define a single one in an appropriate place (cs-client.js), and extend it as you discover more fields. If the tests for a particular file want to provide fake data for a Feature, and you don't want to include all the fields that the API provides, you can define a local subset using `Pick<Feature, 'id'|'name'|'summary'|...>`: https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys. You can define a local alias for that subset using 

```typescript
type AliasName = Pick<...>;
```

But I'd generally avoid exporting those aliases.